x/ibc/24-host/keys.go

```diff
@@ -69,8 +71,8 @@ func ClientTypePath() string {
 
 // ConsensusStatePath takes an Identifier and returns a Path under which to
 // store the consensus state of a client.
-func ConsensusStatePath(height uint64) string {
-	return fmt.Sprintf("consensusState/%d", height)
+func ConsensusStatePath(height exported.Height) string {
+	return fmt.Sprintf("consensusState/%s", height)
```

Comment:
this will be stored as `"<epochNumber>-<epochHeight>"`, right? Do we need to document this on the SDK specification and ICS specs?

Reply:
This is documented in the SDK docs now